### PR TITLE
fix discover e2e

### PR DESCRIPTION
**Pull request by skylarbarrera**

### CODE REVIEW

File: src/components/Spinner.js [hunk 1]
```diff
@@ -8,13 +9,15 @@ import { colors, position } from '@rainbow-me/styles';
...
 const Spinner = ({ color, duration, size, ...props }) => (
   <Centered {...props}>
-    <SpinAnimation duration={duration}>
-      <ImgixImage
-        source={SpinnerImageSource}
-        style={position.sizeAsObject(size)}
-        tintColor={color}
-      />
-    </SpinAnimation>
+    {!IS_TESTING && (
```
Comment (brunobar79) [inline src/components/Spinner.js:12]: Remember that unfortunately this is a string so it should be 
`IS_TESTING !== 'true'`

**Review by brunobar79 — APPROVED** [review]
🙏🙏🙏

File: src/components/Spinner.js (hunk 1 above)
Comment (skylarbarrera) [reply src/components/Spinner.js:12]: true!! i had it right and then i moved into the actual component. will fix pre-merge

**Review by osdnk — APPROVED** [review]
amazing

**Pull request merged**